app/src/main/java/org/mozilla/fenix/library/bookmarks/BookmarkFragment.kt

```diff
@@ -44,13 +46,14 @@ import org.mozilla.fenix.R
 import org.mozilla.fenix.components.FenixSnackbar
 import org.mozilla.fenix.components.StoreProvider
 import org.mozilla.fenix.databinding.FragmentBookmarkBinding
-import org.mozilla.fenix.ext.bookmarkStorage
 import org.mozilla.fenix.ext.components
-import org.mozilla.fenix.ext.minus
-import org.mozilla.fenix.ext.nav
 import org.mozilla.fenix.ext.requireComponents
 import org.mozilla.fenix.ext.setTextColor
+import org.mozilla.fenix.ext.nav
+import org.mozilla.fenix.ext.bookmarkStorage
+import org.mozilla.fenix.ext.minus
 import org.mozilla.fenix.ext.toShortUrl
+import org.mozilla.fenix.ext.getRootView
```

Comment:
There's something weird about these imports. Running the formatter should order them alphabetically. So, I am wondering if you're manually re-ordering things?

Reply:
Nice catch, thanks :)
The default settings for the formatter is to use a wildcard import if there are more then 5 imports. So I imported manually. Changed the rules.